tests/em/static/test_DC_2D_jvecjtvecadj.py

```diff
@@ -280,27 +290,27 @@ def setUp(self):
         self.survey = survey
         self.dmis = dmis
 
-    def test_misfit(self):
-        passed = Tests.checkDerivative(
-            lambda m: (
-                self.survey.dpred(m), lambda mx: self.p.Jvec(self.m0, mx)
-            ),
-            self.m0,
-            plotIt=False,
-            num=3
-        )
-        self.assertTrue(passed)
-
-    def test_adjoint(self):
-        # Adjoint Test
-        # u = np.random.rand(self.mesh.nC*self.survey.nSrc)
-        v = np.random.rand(self.mesh.nC)
-        w = np.random.rand(self.survey.dobs.shape[0])
-        wtJv = w.dot(self.p.Jvec(self.m0, v))
-        vtJtw = v.dot(self.p.Jtvec(self.m0, w))
-        passed = np.abs(wtJv - vtJtw) < 1e-8
-        print('Adjoint Test', np.abs(wtJv - vtJtw), passed)
-        self.assertTrue(passed)
+    # def test_misfit(self):
```

Comment:
did you intend to stop running these tests? or were they just commented out for debugging?

Reply:
oops. nice catch. I was running tests